spec/services/openid_connect_attribute_scoper_spec.rb

```diff
@@ -96,6 +96,25 @@
       end
     end
 
+    context 'with profile:name scope' do
+      let(:scope) { 'openid profile:name' }
+
+      it 'includes name attributes' do
+        expect(filtered[:given_name]).to be_present
```

Comment:
in European-derived cultures, yes. The order (first) is culturally-dependent, ergo "given" instead of "first".

https://en.wikipedia.org/wiki/Given_name